cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -427,21 +427,21 @@ def test_migration_rbac(appliance, new_credential, v2v_providers):
 
 
 @pytest.mark.ignore_stream("5.9")
-@pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs']], indirect=True)
-def test_edit_mapping_description(appliance, v2v_providers, form_data_single_datastore,
-                                    host_creds, conversion_tags, soft_assert):
+def test_edit_mapping_fields(appliance, v2v_providers, _form_data,
```

Comment:
I think here you are using `_form_data` that I referred in comment above. 
**Suggestion**  Can we rather write a new fixture that encapsulates the `_form_data` call instead? That way we leave `_form_data` as it was intended to work/be used.

Reply:
**Required**

`_form_data` itself was not designed to be a fixture, there are other fixtures in `cfme/fixtures/v2v` that use it as a method to generate a base dictionary.